tests/crops_test.py

```diff
@@ -0,0 +1,25 @@
+import warnings
```

Comment:
Let's rename the file, for example `test_crop_and_search.py` or `test_search_real_images.py` will work fine - this way it will be clear that we apply both crop and search here.